types/ext/dictionary.d.ts

```diff
@@ -253,6 +253,15 @@ export type TermDictionaryEntry = {
     frequencies: TermFrequency[];
 };
 
+export type InflectionHypothesis = {
```

Comment:
Is it really a "hypothesis" at this stage, or is this just a grouped/labeled set of inflections?